psychopy/iohub/util/__init__.py

```diff
@@ -367,6 +367,26 @@ def convertCamelToSnake(name, lower_snake=True):
 getCurrentDateTimeString = lambda: getCurrentDateTime().strftime("%Y-%m-%d %H:%M")
 
 
+# rgb255 color utils
+def hilo(a, b, c):
```

Comment:
I'm interested by these functions, could we move them to `psychopy.tools.colorspacetools` & potentially add references to them in methods of `Color`?